docs/management/admin/host-isolation-exceptions.asciidoc

```diff
@@ -6,6 +6,8 @@ You can configure host isolation exceptions for specific IP addresses that <<hos
 
 Host isolation exceptions support IPv4 addresses, with optional classless inter-domain routing (CIDR) notation.
 
+By default, a host isolation exception is recognized globally across all hosts running {endpoint-sec}. If you have a https://www.elastic.co/pricing[Platinum or Enterprise subscription], you can also assign a host isolation exception to a specific {endpoint-sec} integration policy, affecting only the hosts assigned to that policy.
```

Comment:
Slight difference with the Host Isolation exceptionlist artifact - the entire host isolation feature is platinum/enterprise, so even having a global list is in a paid tier. 

(Realizing we don't say on the Host Isolation page that it is a paid feature, so that may be good to add separately!)